.eslintrc.js

```diff
@@ -88,6 +88,8 @@ module.exports = {
 
         // We need to test things without type guards sometimes
         "@typescript-eslint/no-unsafe-assignment": "off",
+        "@typescript-eslint/no-unsafe-call": "off",
+        "@typescript-eslint/consistent-type-assertions": "off",
```

Comment:
turning both of these off seems rather aggressive. i'd rather see individual `eslint-disable` comments for the areas that we can't fix which violate these.

Reply:
ah this is tests, nevermind